### DISCUSSION TITLE
GitHub code view looks dark and blurry on my Chrome

### DISCUSSION THREAD

**Discussion by Cyl18** [community/community#118646] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

it looks like this:
![image](https://github.com/community/community/assets/14993992/1e373695-9a02-4fb3-b148-c5f2c802f857)

**Comment by eliz-amst** [comment]
Check Browser Zoom Settings:
Ensure that your browser zoom settings are set to 100%. Zooming in or out can sometimes cause text and images to appear blurry. You can reset the zoom level by pressing Ctrl + 0 (Windows/Linux) or Cmd + 0 (Mac).
Clear Browser Cache and Cookies:
Clearing your browser's cache and cookies can help resolve display issues. Go to Chrome Settings > Privacy and security > Clear browsing data. Select "Cached images and files" and "Cookies and other site data," then click "Clear data."
Disable Chrome Extensions:
Browser extensions can sometimes interfere with website rendering. Try disabling any extensions you have installed, then reload GitHub to see if the issue persists. If the problem is resolved, re-enable your extensions one by one to identify the culprit.
Update Chrome:
Ensure that your Chrome browser is up to date. Go to Chrome Settings > About Chrome and check for updates. If updates are available, install them and restart your browser.
Check Display Settings:
Verify your computer's display settings to ensure they are configured correctly. Check the display resolution and scaling settings to see if they might be affecting the clarity of text and images.
Test in Incognito Mode:
Open GitHub in Chrome's Incognito mode to see if the issue persists. This will help determine if the problem is caused by browser extensions or cache.
Try a Different Browser:
Test GitHub in a different browser (e.g., Firefox, Edge) to see if the issue is specific to Chrome. If GitHub displays properly in another browser, the problem may be related to Chrome settings or compatibility.
Contact GitHub Support:
If the issue persists after trying these troubleshooting steps, consider reaching out to GitHub Support for further assistance.

**Comment by Cyl18** [comment, marked as answer]
found it, it comes from a plugin called `Font Rendering Enhancer`
